reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -316,6 +318,9 @@ def create_hparams(flags):
         save_model=flags["save_model"] if "save_model" in flags else True,
         save_epoch=flags["save_epoch"] if "save_epoch" in flags else 5,
         metrics=flags["metrics"] if "metrics" in flags else None,
+        ranking_metrics=flags["ranking_metrics"] if "ranking_metrics" in flags else None,
+        user_item_file= flags["user_item_file"] if "user_item_file" in flags else None,
+        top_K=flags["top_K"] if "top_K" in flags else None,
         write_tfevents=flags["write_tfevents"] if "write_tfevents" in flags else False,
     )
 
```

Comment:
the hparams part of TF always makes me shudder.

there's a cleaner way to pull values from dictionaries with default values, it would be good to clean up this section to something like this:

```
save_epoch = flags.get('save_epoch', 5)
metrics = flags.get('metrics')
```